docs/deployment/managed-scanning/azure.md

```diff
@@ -0,0 +1,121 @@
+---
+slug: azure
+append_help_link: true
+title: Azure DevOps
+hide_title: true
+description: Learn how to add an Azure DevOps repository to Semgrep Managed Scans.
+tags:
+  - Beta
+  - Deployment
+  - Semgrep AppSec Platform
+---
+
+import ScanWithSms from "/src/components/procedure/_scan-with-sms.mdx"
+
+# Add an Azure DevOps repository to Semgrep Managed Scans
+
+Add Azure DevOps repositories to your Semgrep organization in bulk without adding or changing your existing CI workflows through **Managed Scans**. 
+
+
+## Prerequisites and permissions
+
+Semgrep Managed Scanning requires one of the following plans:
+
+//TODO
+
+You must provide the following tokens with the specified scopes...
```

Comment:
What scopes are required?

Reply:
Owner or Project Collection Admin according to the [prerequisites needed to configure webhooks](https://learn.microsoft.com/en-us/azure/devops/service-hooks/services/webhooks?toc=%2Fazure%2Fdevops%2Fmarketplace-extensibility%2Ftoc.json&view=azure-devops#prerequisites)